std/assembly/util/sort.ts

```diff
@@ -32,10 +32,11 @@ export function COMPARATOR<T>(): (a: T, b: T) => i32 {
       if (a === b || a === null || b === null) return 0;
       var alen = changetype<string>(a).length;
       var blen = changetype<string>(b).length;
-      if (!alen && !blen) return 0;
+      if ((alen | blen) == 0) return 0;
```

Comment:
Minor suggestion for patterns like these:
```suggestion
      if (!(alen | blen)) return 0;
```
A little more compact to parse, and unary `i32.eqz` with no constant right away so a little lighter to optimize. Probably not super important, more a case of "why not" when there is the opportunity :)